homeassistant/components/samsungtv/bridge.py

```diff
@@ -223,9 +224,12 @@ def try_connect(self):
                 return RESULT_SUCCESS
             except WebSocketException:
                 LOGGER.debug("Working but unsupported config: %s", config)
-                return RESULT_NOT_SUPPORTED
+                result = RESULT_NOT_SUPPORTED
             except (OSError, ConnectionFailure) as err:
                 LOGGER.debug("Failing config: %s, error: %s", config, err)
+        else:
```

Comment:
I think we can disable that pylint warning. The `return` statements also break the loop.